src/GFDL-1.1-or-later.xml

```diff
@@ -5,16 +5,6 @@
     <crossRefs>
       <crossRef>http://www.gnu.org/licenses/old-licenses/fdl-1.1.txt</crossRef>
     </crossRefs>
-    <standardLicenseHeader>
-      Copyright (c) <alt name="copyright" match=".+">YEAR YOUR NAME</alt>
-      Permission is granted to copy, distribute and/or modify this
-      document under the terms of the GNU Free Documentation License,
-      Version 1.1 or any later version published by the Free Software
-      Foundation; with the Invariant Sections being LIST THEIR
-      TITLES, with the Front-Cover Texts being LIST, and with the
```

Comment:
“LIST THEIR TITLES”, etc. are markers for variable text.  In the new `<standardLicenseHeader>` below, I've marked those up with `<alt>.  This same issue applies to the other GFDL versions too.

Reply:
Check the formatting on your comment above - missing some text

Reply:
> Check the formatting on your comment above - missing some text

Thanks, fixed.